### PR TITLE
PointCloudTool support multi selection

### PR DESCRIPTION
**Pull request by Glenfiddish**

### CODE REVIEW

File: packages/lb-annotation/src/core/toolOperation/pointCloud2dOperation.ts [hunk 1]
```diff
@@ -143,6 +160,32 @@ class PointCloud2dOperation extends PolygonOperation {
     return PolygonUtils.getPolygonByIDs(this.polygonList, this.selectedIDs);
   }
 
+  public updateSelectedPolygonsPoints(offset: Partial<ICoordinate>) {
+    if (this.selectedPolygons && this.selectedPolygons?.length > 0) {
+      const originPolygonList = _.cloneDeep(this.selectedPolygons!);
+      const updateList: UpdatePolygonByDragList = [];
+
+      this.selectedPolygons?.forEach((polygon, index) => {
+        polygon.pointList = polygon.pointList.map((point) => {
```
Comment (Kerwin-L) [inline packages/lb-annotation/src/core/toolOperation/pointCloud2dOperation.ts:169]: This way will change the origin polygonList in `this.polygonList`. Creating a new PointList is better.

File: packages/lb-components/src/views/MainView/toolFooter/AnnotatedAttributes/index.tsx [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+export const AnnotatedAttributesPanel = () => {
+  const stepConfig: IPointCloudConfig = useSelector(stepConfigSelector);
+  const { attrPanelLayout, setAttrPanelLayout } = useContext(PointCloudContext);
+
+  return (
+    <div className={getClassName('annotated-attribute')}>
+      {attrPanelLayout ? (
+        <div className={getClassName('annotated-attribute', 'text')}>
+          <span>标注结果</span>
```
Comment (Kerwin-L) [inline packages/lb-components/src/views/MainView/toolFooter/AnnotatedAttributes/index.tsx:100]: 后续有空的时候支持下 I18n?

File: packages/lb-components/src/components/pointCloudView/PointCloudBackView.tsx [hunk 3]
```diff
@@ -137,8 +223,15 @@ const PointCloudSideView = ({ currentData, config, checkMode }: IA2MapStateProps
...
     backPointCloudPolygonOperation.singleOn(
       'updatePolygonByDrag',
-      ({ newPolygon, originPolygon }: any) => {
-        backViewUpdateBox?.(newPolygon, originPolygon)
+      (updateList: UpdatePolygonByDragList) => {
+        if (ptCtx.selectedIDs.length === 1 && updateList.length === 1) {
+          const { newPolygon, originPolygon } = updateList[0];
+
+          if (newPolygon && originPolygon) {
+            transferPolygonDataToBoxParams(newPolygon, originPolygon);
```
Comment (Kerwin-L) [inline packages/lb-components/src/components/pointCloudView/PointCloudBackView.tsx:231]: What does the function do?

Comment (Glenfiddish) [reply packages/lb-components/src/components/pointCloudView/PointCloudBackView.tsx:231]: transfer PolygonData To BoxParams and rendering new box

File: packages/lb-annotation/src/core/toolOperation/pointCloud2dOperation.ts (hunk 1 above)
Comment (Glenfiddish) [reply packages/lb-annotation/src/core/toolOperation/pointCloud2dOperation.ts:169]: It need to be change pointList

**Pull request merged**